opentelemetry-userevents-exporter/src/logs/exporter.rs

```diff
@@ -0,0 +1,252 @@
+#![allow(unused_imports, unused_mut, unused_variables, dead_code)]
+
+use eventheader::{FieldFormat, Level, Opcode};
+use eventheader_dynamic::{EventBuilder, EventSet};
+//use crate::exporter_traits::*;
+use async_trait::async_trait;
+
+use std::any::Any;
+use std::fs::File;
+use std::sync::Arc;
+use std::fmt::Debug;
+use std::borrow::Cow;
+use chrono::{Datelike, Timelike};
+
+
+use std::{cell::RefCell, time::SystemTime, str};
+use opentelemetry_api::{logs::Severity, logs::AnyValue, Array, Key, Value};
+
+pub type ProviderGroup = Option<Cow<'static, str>>;
+
+thread_local! { static EBW: RefCell<EventBuilder> = RefCell::new(EventBuilder::new());}
+
+pub struct ExporterConfig{
+    pub keyword: u64
+}
+
+impl Default for ExporterConfig {
+    fn default() -> Self {
+        ExporterConfig { keyword: 1}
+    }
+}
+
+impl ExporterConfig {
+    pub(crate) fn get_log_event_keyword(&self) -> u64{
+        self.keyword
+    }
+}
+
+pub(crate) struct UserEventsExporter {
+    provider: Arc<eventheader_dynamic::Provider>,
+    exporter_config: ExporterConfig,
+}
+
+//TBD - How to configure provider name and provider group
+impl UserEventsExporter {
+    pub(crate) fn new(
+        provider_name: &str,
+        provider_group: ProviderGroup,
+        exporter_config: ExporterConfig,
+    ) -> Self
+    {
+        let mut options = eventheader_dynamic::Provider::new_options();
+        options = *options.group_name(provider_name);
+        let mut provider: eventheader_dynamic::Provider = eventheader_dynamic::Provider::new(provider_name, &options);
```

Comment:
```suggestion
        let mut eventheader_provider: eventheader_dynamic::Provider = eventheader_dynamic::Provider::new(provider_name, &options);
```

Reply:
nit to help to avoid confusing "provider", as Otel also has provider concept

Reply:
Good point. Done.